tobac/utils/general.py

```diff
@@ -551,3 +556,146 @@ def combine_tobac_feats(list_of_feats, preserve_old_feat_nums=None):
     combined_sorted["feature"] = np.arange(1, len(combined_sorted) + 1)
     combined_sorted = combined_sorted.reset_index(drop=True)
     return combined_sorted
+
+
+@internal_utils.irispandas_to_xarray
+def transform_feature_points(
+    features,
+    new_dataset,
+    latitude_name="auto",
+    longitude_name="auto",
+    altitude_name="auto",
+    max_time_away=datetime.timedelta(minutes=0),
+    max_space_away=0,
+    max_vspace_away=0,
```

Comment:
This is a good point, and we typically try to avoid surprising user behavior. I've switched these to `None` and have added the appropriate logic to allow for no time/space maximum to be used.